brainscore_vision/nwb_integration/dandi_to_stimulus_set.py

```diff
@@ -0,0 +1,130 @@
+import os, re
+import logging
+
+from brainio.stimuli import StimulusSet
+from dandi.dandiapi import DandiAPIClient
+from pathlib import Path
+from PIL import Image
+from pynwb.file import NWBFile
+from tqdm import tqdm
+from typing import Tuple
+
+
+logger = logging.getLogger(__name__)
+
+def extract_number(filename: str) -> int: 
+    # Extract the number from the filename and return it as an integer
```

Comment:
I like the style of comments in the assembly file, if you wanted to keep them consistent then it might be worth it to change the format of the function comments